### DISCUSSION TITLE
OrchardCore.Workflows example recipes

### DISCUSSION THREAD

**Discussion by lampersky** [OrchardCMS/OrchardCore#11516] [Show and tell]
for all those who are struggling with **OrchardCore.Workflows** module I've prepared a few examples, which shows, how to use events and tasks, and how to connect them together

here you can find [recipes](https://github.com/lampersky/OrchardCoreWorkflowsExamples#simple-blog) and some demo videos

enjoy!

**Comment by hishamco** [comment]
Can we add some to the How to ..? in our docs, please have a look to the workflow docs and check if we need to add few examples

Thanks

**Comment by lampersky** [comment]
I've just added another example - simple **WordPress** Importer Application. This one is a bit more complicated.

You can find recipe here:
https://github.com/lampersky/OrchardCoreWorkflowsExamples/blob/main/WorkflowWordpressImport.json
Guide with steps:
https://github.com/lampersky/OrchardCoreWorkflowsExamples/blob/main/WorkflowWordpressImport.md

Also you can watch app in action here:
https://www.youtube.com/watch?v=dLPNuBQ7yKQ

**Reply by hishamco** [reply]
It's great to see a demo in OC standup

**Comment by giannik** [comment]
you are a workflows guru!

**Comment by sebastienros** [comment]
Weird that it's a workflow but maybe this way users can customize it. Why not create parallel workflows to create the content types, or just make a module that would create them, have the views, and a workflow definition that is customizable?

**Reply by lampersky** [reply]
@sebastienros 
It was created just to show how powerful workflow module really is, of course dedicated import module would be better, but this way you can also import everything you want, and you can slightly adjust it to your needs without any recompilation etc.

It is a part of my Workflow examples, I've made it for fun and currently I'm adding media import. Once it's done, we will have nice background to start thinking about dedicated import module. To be honest, I'm not a Wordpress guy and I don't know how real import should look like. But I think, somebody interested in this topic, can make use of it, to perform some basic import.
